### DISCUSSION TITLE
Not able to use `pnpm` to create a SvelteKit project

### DISCUSSION THREAD

**Discussion by matiadev** [sveltejs/kit#4797] [Q&A] [answered]
I tried making a project using `pnpm init svelte` but it just creates a `package.json` file as described in the [pnpm documentation](https://pnpm.io/cli/init).

I'm wondering is this intended or something worth reporting as an issue? I used `pnpm` before and it feels weird I have to use `npm` to initialize the project before using `pnpm` to install dependencies.

Thank you!

**Comment by bluwy** [comment, marked as answer]
You should use `pnpm create svelte` instead

**Reply by matiadev** [reply]
> You should use `pnpm create svelte` instead

Thank you! 😄

I was wondering what changed but now I remember updating `pnpm` so something changed since the last time I created a project which makes a lot of sense.

**Reply by mrkishi** [reply]
`pnpm init` used to invoke `npm init`. It's now its own command with no "initialize from template" features as those are handled by `pnpm create`.

**Reply by matiadev** [reply]
> `pnpm init` used to invoke `npm init`. It's now its own command with no "initialize from template" features as those are handled by `pnpm create`.

Thank you! I've read about it but I wasn't sure.

**Reply by marufmax** [reply]
um ? 
<img width="429" alt="image" src="https://github.com/user-attachments/assets/cfcffd1e-c9cb-4666-b5ea-bacb1c40712f" />

**Reply by ghostdevv** [reply]
Yea this is outdated now @marufmax. You now should run `pnpm dlx sv create` (or `pnpx` if available/preferred). See: https://svelte.dev/blog/sv-the-svelte-cli
